.github/workflows/maven-cd.yml

```diff
@@ -88,7 +88,7 @@ jobs:
       # https://github.com/jenkins-infra/github-reusable-workflows/issues/36
       - name: Set up Maven
         run: |
-          wget --no-verbose https://downloads.apache.org/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
+          wget --no-verbose https://dlcdn.apache.org/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
```

Comment:
or 
```
https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/$MAVEN_VERSION/apache-maven-$MAVEN_VERSION-bin.zip
```

Reply:
Since dlcdn.apache.org is the download URL provided by https://maven.apache.org/download.cgi , I think it is the better choice than using the Apache artifact repository.

Reply:
FTR  `https://repo.maven.apache.org/maven2` is the URL provided by default by Apache Maven distribution in the super pom.  So, it's the URL used by any default Apache Maven distribution.
It's a CNAME to Maven Central. 
If you prefer using something else, that is fine, but Maven Central will scale more and be more reliable than Apache Mirrors which can fail and even be outdated.